redaxo/src/addons/install/lib/package/package_download.php

```diff
@@ -14,7 +14,7 @@ abstract class rex_install_package_download
 
     public function run(string $addonkey, int $fileId)
     {
-        $this->addonkey = $addonkey;
+        $this->addonkey = rex_path::basename($addonkey); // the addonkey is used in file paths
```

Comment:
Aktuell ist der Code eigentlich sicher (aus meiner Sicht) wegen der exception in zeile 22.
Das ist aber auch davon abhängig, ob redaxo.org korrekt abgesichert ist.
Daher denke ich, ist dieser zusätzliche Schutz hier nicht verkehrt.
Und so ist es für psalm auch als sicher erkennbar.